__tests__/api.test.js

```diff
@@ -33,4 +39,81 @@ describe("GET /api/topics", () => {
         .expect(404)
     })
   })
-})
\ No newline at end of file
+}),
+
+describe('GET api/articles', () => {
+
+}),
+
+// Test 1 - api page returns with an object
```

Comment:
Please remove all comments and console logs before committing 👍

Reply:
> Please remove all comments and console logs before committing 👍

Would it be good practice to leave the comments in explaining how the tests for the length is working? I understand deleting the test 1 etc. but am just curious as to whether I should keep this bit in or not.